pkg/kubelet/kubelet.go

```diff
@@ -1521,6 +1545,12 @@ func (kl *Kubelet) tryUpdateNodeStatus() error {
 		return fmt.Errorf("no node instance returned for %q", kl.hostname)
 	}
 
+	if kl.configureCBR0 {
```

Comment:
The intent is to let it be set to false for other node platforms (e.g. aws, vagrant, coreos, etc) but enable this unconditionally for gce/debian.